navit/android/src/org/navitproject/navit/NavitGraphics.java

```diff
@@ -37,6 +40,7 @@
 import android.os.Message;
 import android.support.v4.view.ViewConfigurationCompat;
 import android.util.Log;
+import android.net.Uri;
```

Comment:
Build fails due to checkstyle is failing:
[ant:checkstyle] [ERROR] /home/circleci/code/navit/android/src/org/navitproject/navit/NavitGraphics.java:43: Wrong lexicographical order for 'android.net.Uri' import. Should be before 'android.util.Log'. [CustomImportOrder]

Could you please fix this?

Reply:
Yes, sure!

Reply:
OK, done!